engine/classes/ElggFile.php

```diff
@@ -18,6 +18,10 @@
  *
  * @package    Elgg.Core
  * @subpackage DataModel.File
+ *
+ * @property string $thumbnail  Path to small thumbnail relative to file owner's filestore directory
+ * @property string $smallthumb Path to medium thumbnail relative to file owner\'s filestore directory
+ * @property string $largethumb Path to large/master thumbnail relative to file owner\'s filestore diretory
```

Comment:
Do we care for these here? I personally never remember, so hinting is nice, but it is also irrelevant to most files (especially non entities).